Minor issue. Patch can be submitted upstream
Index: bpfcc/tools/CMakeLists.txt
===================================================================
--- bpfcc.orig/tools/CMakeLists.txt	2017-02-22 15:11:07.490076012 +0530
+++ bpfcc/tools/CMakeLists.txt	2017-02-22 15:11:07.483409166 +0530
@@ -7,5 +7,4 @@
   install(PROGRAMS ${FIL} DESTINATION share/bcc/tools RENAME ${FIL_WE})
 endforeach()
 install(FILES ${C_FILES} DESTINATION share/bcc/tools)
-install(FILES ${TXT_FILES} DESTINATION share/bcc/tools/doc)
-add_subdirectory(old)
+install(FILES ${TXT_FILES} DESTINATION share/bcc/doc)
